test/fork/run-mac-only.test.js

```diff
@@ -9,7 +9,7 @@ const filenames = [
   [__dirname + 'some\ \\file', '#!/bin/sh\necho "OK"'],
 ];
 
-if (!process.env.TRAVIS && process.platform === 'darwin') {
+if (false && !process.env.TRAVIS && process.platform === 'darwin') {
```

Comment:
I assume this is not intended to stay… 😉

Reply:
I'm going to remove those tests altogether. They don't run in travis (intentionally) and they're messing with local tests. The tests will pass and fail at random on a mac, so they're not doing their job properly.